microsoftgraph/microsoft-graph-explorer-v4
# Feature: filter permissions by consented state

## Overview

Fixes #2417 
Enables filtering based on the status column so that a user can easily see the consented permissions vs unconsented permissions

### Demo

![image](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/assets/58787602/d55bac25-4843-4102-a561-331cb2b86f40)

![image](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/assets/58787602/84901001-57eb-46b5-b0cc-6a6a50894a26)


## Testing Instructions

* Open all permissions
* Click on the filter button next to the search bar
* Notice items are filtered by their consent status
